src/core/evaluator.js

```diff
@@ -335,12 +335,16 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
         });
         imgData.cached = true;
```

Comment:
Note for possible follow-up: This really doesn't make sense, since whether or not caching will actually happen depends on the existence of a `cacheKey`; should this be `imgData.cached = !!cacheKey; instead?